src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/template.json

```diff
@@ -26,31 +26,6 @@
   "sources": [
     {
       "modifiers": [
-        {
-          "condition": "(!OrganizationalAuth && !IndividualAuth)",
-          "exclude": [
-            "Areas/Identity/Pages/Shared/_LoginPartial.Identity.cshtml",
-            "Areas/Identity/Pages/Shared/_LoginPartial.OrgAuth.cshtml"
-          ]
-        },
```

Comment:
How come it's legitimate to remove this (or the updated equivalent)? Don't we still need to exclude `_LoginPartial.cshtml` in the case where you're not using auth at all?

I'm surprised that no test updates are needed in this PR. Is it because `Template_Produces_The_Right_Set_Of_FilesAsync` is quarantined? If that's the reason, could you make sure you've run the template E2E tests locally and verified they pass? Thanks!

Reply:
Actually I checked the build logs and somehow `Template_Produces_The_Right_Set_Of_FilesAsync` appears to have passed. I don't get it - doesn't that test verify that `_LoginPartial.cshtml` should *not* be present in the no-auth case?

Reply:
Ah never mind. I see that a higher-level directory gets excluded in the no-auth case:

```
        {
          "condition": "(!IndividualLocalAuth)",
          "exclude": [
            "Data/SqlLite/**",
            "Data/SqlServer/**",
            "Data/ApplicationDbContext.cs",
            "Areas/**"
          ]
        },
```

This explains why it's all still working! Sorry for the confusion.